package.json

```diff
@@ -48,12 +50,14 @@
     "eslint-plugin-vitest": "^0.3.10",
     "eslint-plugin-vue": "9.28.0",
     "eslint-plugin-vuetify": "2.4.0",
+    "jest": "^29.7.0",
     "prettier": "3.3.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "rimraf": "6.0.1",
     "sass": "1.79.4",
     "storybook": "7.6.0",
+    "typescript": "^5.7.2",
```

Comment:
I very much want to get typescript up and running for Cuttle but that is a separate project and I'd prefer to keep updates minimally scoped to the essential changes for a given issue, so lets leave dependency changes out of this PR